Lib/test/regrtest.py

```diff
@@ -533,9 +538,17 @@ def work():
                                    universal_newlines=True,
                                    close_fds=(os.name != 'nt'))
                     stdout, stderr = popen.communicate()
+                    retcode = popen.wait()
+
                     # Strip last refcount output line if it exists, since it
                     # comes from the shutdown of the interpreter in the subcommand.
                     stderr = debug_output_pat.sub("", stderr)
+
+                    if retcode != 0:
```

Comment:
Oh... it's a bug in master. If the process crashed, we must not modify stdout but dump it unchanged!